model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -576,7 +581,13 @@ service TestStreamService {
   // A TestStream will request for events using this RPC.
   rpc Events(EventsRequest) returns (stream TestStreamPayload.Event) {}
 }
-message EventsRequest {}
+
+message EventsRequest {
+  // The set of tags to read from. These tags are a subset of the
+  // TestStreamPayload's output_tags. This allows Interactive Beam to cache
+  // many PCollections from a pipeline then replay a subset of them.
+  repeated string tags = 1;
```

Comment:
The usage of `tag` and `tags` is a poor name choice. I would suggest replacing them with `output_id` and `output_ids` and please update the comments to refer to the [PTransform outputs local names](https://github.com/apache/beam/blob/dd9f5d28cc2f55697283058963fd2d8607aa3cd6/model/pipeline/src/main/proto/beam_runner_api.proto#L165).

Reply:
Updated to output_ids and modified the comment.